### PR TITLE
the edit link on the delete place confirm page was broken

### PR DESCRIPTION
**Pull request by wheelsandcogs**
`settings.page` was undefined - just pass the actual route string instead.

### CODE REVIEW

File: pages/place/routers/confirm.js [hunk 1]
```diff
@@ -71,11 +71,11 @@ module.exports = settings => {
       if (req.session.form && req.session.form[req.model.id]) {
         return next();
       }
-      return res.redirect(req.buildRoute(settings.page));
+      return res.redirect(req.buildRoute('place.delete'));
     },
     editAnswers: (req, res, next) => {
       delete req.session.form[req.model.id].validationErrors;
-      return res.redirect(req.buildRoute(settings.page));
+      return res.redirect(req.buildRoute('place.delete'));
```
Comment (lennym) [inline pages/place/routers/confirm.js:78]: This feels too specifically "delete" for a router which is not specifically "delete".

Comment (wheelsandcogs) [reply pages/place/routers/confirm.js:78]: Good point - will find a proper solution

File: pages/place/delete/index.js [hunk 2]
```diff
@@ -44,6 +44,7 @@ module.exports = settings => {
 
   app.use('/confirm', (req, res, next) => {
     confirm({
+      page: 'place.delete',
```
Comment (wheelsandcogs) [inline pages/place/delete/index.js:47]: This is the bug fix, everything else is just tidying up issues I noticed while testing.

File: pages/place/routers/confirm.js [hunk 3]
```diff
@@ -71,7 +71,7 @@ module.exports = settings => {
       if (req.session.form && req.session.form[req.model.id]) {
         return next();
       }
-      return res.redirect(req.buildRoute(settings.page));
+      return res.redirect(req.buildRoute('place.delete'));
```
Comment (lennym) [inline pages/place/routers/confirm.js:74]: This looks like it possibly needs reverting.

Comment (wheelsandcogs) [reply pages/place/routers/confirm.js]: was just testing you were awake 😄

**Pull request merged**